### PR TITLE
KEYCLOAK-17330 Document how to use private extensions

### PR DESCRIPTION
**Pull request by apeschel**
Relies on https://github.com/keycloak/keycloak-operator/pull/340

Adds instructions on how to use HTTP basic auth URLs to access private extensions.

### CODE REVIEW

**Review by vmuzikar — COMMENTED** [review]
@iankko Could you please confirm RH-SSO init container officially supports HTTP auth?

This needs to be merged after https://github.com/keycloak/keycloak-operator/pull/389.

**Review by andymunro — APPROVED** [review]

**Comment by andymunro** [comment]
@apeschel To fix the Waiting for Status problem, use this command: git pull upstream main

**Review by andymunro — CHANGES_REQUESTED** [review]
To fix the Waiting for Status problem, use this command: git pull upstream main

**Comment by apeschel** [comment]
@andymunro Just updated it

**Review by andymunro — COMMENTED** [review]
Thanks, @apeschel.  I am not sure why you are still getting the message about Waiting for status ..  Usually it is fixed by:  git pull upstream main

@pskopek Do you have any ideas?

**Comment by apeschel** [comment]
@andymunro On my side it says it's waiting for a maintainer to approve the run. Just updated it again.

**Comment by ahus1** [comment]
Hi @iankko - there is still an open question on this PR. Can you answer this question, or should this PR rather be closed? Thanks!

> @iankko Could you please confirm RH-SSO init container officially supports HTTP auth?

**Comment by ahus1** [comment]
@iankko - update: I'm closing this PR as it relates to a PR for the old operator which has not been merged.

**Pull request closed without merging**
